.github/workflows/rust.yml

```diff
@@ -192,7 +192,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- --verbose
+          args: -- -D warnings --verbose
```

Comment:
What does this do?

Reply:
> What does this do?

It makes all clippy warnings into errors, so even warnings don't pass through the CI

Reply:
So, this will still fail the CI even if we put everything in clippy as a warning, right? Then, I would leave `clippy::all` as `deny`.

Reply:
Yes. lets leave it as is.